test_app/src/index.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+import ReactDOM from 'react-dom/client';
+
+import TwoPiRest from '../../lib/dist';
+
+const elem = document.getElementById("App") as HTMLElement;
+if (elem) {
```

Comment:
For some weird reason, I am not able to typecast this var. need to check in tsconfig but for this work around is working.